Features/Language.feature

```diff
@@ -0,0 +1,34 @@
+Feature:Language Feature
+
+Scenario Outline:1- user is able to add a language and choose a level and save
+Given enters valid username '<username>' and password '<password>'  clicks login button 
+When user naviagate to language tab and click Add new button
+And User adds a language '<language>' with level '<level>' clicks  Add button
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).